main.go

```diff
@@ -50,6 +51,7 @@ func main() {
 	root.AddCommand(status.NewCommand())
 	root.AddCommand(unsave.NewCommand())
 	root.AddCommand(update.NewCommand())
+	root.AddCommand(playlist.NewCommand())
```

Comment:
Looks like maybe you forgot to `git add` the new files!

Reply:
🤦